sv-comp/benchexec/my-bench/goblint.xml

```diff
@@ -11,7 +11,7 @@
   <option name="--enable">ana.int.interval</option>
   <!-- <option name="-sets solver td3"/> -->
   <option name="--enable">ana.context.widen</option>
-  <option name="--enable">ana.base.partition-arrays.enabled</option>
+  <option name="-sets exp.arrays-domain partitioned"/>
```

Comment:
This doesn't actually matter. This file isn't used for anything any more and BenchExec doesn't support setting three-argument options from XML, so this choice would be done by the svcomp conf files anyway.

Reply:
Fixed on d0f99ffeec2abff3b5e230d32268be769a5c3696